### PR TITLE
Nept 1440: Removal of the feature likedislike

### PR DESCRIPTION
**Pull request by Fefaine**
### Description

Remove outdated feature likedislike

### Change log

- Removed: Feature like dislike and symlinks in profile

### Commands

```sh
no commands. 
```

### CODE REVIEW

File: profiles/common/modules/custom/multisite_cookie_consent/multisite_cookie_consent.module [hunk 1]
```diff
@@ -50,8 +50,7 @@ function multisite_cookie_consent_init() {
   if (current_path() != 'cookie-notice') {
     // Notice url.
     $notice_page = url('cookie-notice', array('absolute' => TRUE));
-    // Add for example external.
-    drupal_add_js('http://ec.europa.eu/wel/like/js/likedislike.js', 'external');
```
Comment (voidtek) [inline profiles/common/modules/custom/multisite_cookie_consent/multisite_cookie_consent.module]: We cannot remove this line, we should change the link.
for example remplace by http://europa.eu/webtools/load.js

Comment (Fefaine) [reply profiles/common/modules/custom/multisite_cookie_consent/multisite_cookie_consent.module]: Why do you want to replace this line? This feature no longer exists, so the js is no longer needed.

Comment (Fefaine) [reply profiles/common/modules/custom/multisite_cookie_consent/multisite_cookie_consent.module]: Module was removed from the platform

**Comment by imanoleguskiza** [comment]
👍

**Pull request merged**